rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -2523,16 +2561,15 @@ namespace rsx
 		{
 			static inline void impl()
 			{
-				methods[Id] = &T<Index>::impl;
-				bind_range_impl_t<Id + Step, Step, Count, T, Index + 1>::impl();
-			}
-		};
+				if constexpr (Count > 0)
```

Comment:
add static assert if count == 0 while you're at it

Reply:
Done, also added useful asserts about range of methods specified.